tests/models/test_documents.py

```diff
@@ -117,6 +117,16 @@ def test_judgment_content_as_xml(self, mock_api_client):
             "test/1234", show_unpublished=True
         )
 
+    def test_judgment_content_as_xml_tree(self, mock_api_client):
+        mock_api_client.get_judgment_xml_bytestring.return_value = (
+            b'<?xml version="1.0" encoding="UTF-8"?><xml></xml>'
+        )
+        try:
+            document = Document("test/1234", mock_api_client)
+            document.content_as_xml_tree
+        except ValueError as ex:
```

Comment:
What is this test is trying to demonstrate here, that calling document.content_as_xml_tree raises a ValueError?

If so, you can use pytest.raises instead of try, except.

```
mock_api_client.get_judgment_xml_bytestring.return_value = (
           b'<?xml version="1.0" encoding="UTF-8"?><xml></xml>'
)
document = Document("test/1234", mock_api_client)
with pytest.raises(ValueError):
           document.content_as_xml_tree
```

Reply:
aha we *don't* expect it, that's the point - we're asserting that that error is *not* raised!

Reply:
aha ace, will amend!